### PR TITLE
add Juraj Michalek entry, plus review of Omar's book

### PR DESCRIPTION
**Pull request by georgik**
Add author entry, plus reivew of Omar's book

### CODE REVIEW

**Comment by pedrominatel** [comment]
LGTM, @georgik

File: content/blog/review/books-simplified-embedded-rust/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: "Simplified Embedded Rust: A Comprehensive Guide to Embedded Rust Development"
+date: 2024-06-07
+description: "A review of Omar Hiari's books 'Simplified Embedded Rust: ESP Core Library Edition' and 'ESP Standard Library Edition' which offer comprehensive guidance on using Rust for embedded development on Espressif chips."
+author: "Juraj Michálek"
```
Comment (pedrominatel) [inline content/blog/review/books-simplified-embedded-rust/index.md:5]: Change to:

```
showAuthor: false
authors:
    - "juraj-michalek"
```

File: content/blog/review/books-simplified-embedded-rust/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
```
Comment (pedrominatel) [inline content/blog/review/books-simplified-embedded-rust/index.md:1]: Add a cover image, named `featured.png` on the same level as the `index.md`.
The image size could be: 1200x600px

**Comment by pedrominatel** [comment]
Thank you, @georgik! You can remove the folder `review` and keep it as `content/blog/books-simplified-embedded-rust`.

File: data/authors/juraj-michalek.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (pedrominatel) [inline data/authors/juraj-michalek.json:1]: Not mandatory, but you can consider adding a profile picture.

File: content/blog/review/books-simplified-embedded-rust/index.md (hunk 2 above)
Comment (georgik) [reply content/blog/review/books-simplified-embedded-rust/index.md]: Added webp

**Comment by f-hollow** [comment]
@georgik LGTM! Feel free to merge this branch yourself.

**Review by f-hollow — APPROVED** [review]

**Pull request merged**